### PR TITLE
[SecurityBundle] Fix collecting traceable listeners info on lazy firewalls

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Before:
Screenshot 2020-01-07 at 05 17 47

After:
Screenshot 2020-01-07 at 05 18 12

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
I think this breaks laziness of the listeners: `LazyFirewallContext` won't see `$listener instanceof AbstractListener` anymore, isn't it?

**Comment by chalasr** [comment]
@nicolas-grekas ofc you're right. Should be better now

File: src/Symfony/Bundle/SecurityBundle/Debug/WrappedLazyListener.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        $this->response = $event->getResponse();
+
+        return $ret;
+    }
+
+    public function getInfo(): array
+    {
+        if (null !== $this->stub) {
+            // no-op
+        } elseif (self::$hasVarDumper ?? self::$hasVarDumper = class_exists(ClassStub::class)) {
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/SecurityBundle/Debug/WrappedLazyListener.php]: we can even remove this check as var-dumper is a mandatory transitive deps now (same in WrappedListener)

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Bundle/SecurityBundle/Debug/WrappedLazyListener.php (hunk 1 above)
Comment (chalasr) [reply src/Symfony/Bundle/SecurityBundle/Debug/WrappedLazyListener.php]: done

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**